validation/cross_validate_rust_python.py

```
@@ -104,9 +104,14 @@ def carreau_yasuda_python(shear_rate):
     print(f"  Method: apparent_viscosity(shear_rate)")
     
-    # Try to test if we can create a blood model
-    # Note: This depends on cfd_python API structure
-    print(f"\n  TODO: Add cfd_python API test if blood model is exposed")
+    print(f"\n  Cross-checking Python vs Rust:")
+    blood_model = cfd_python.CarreauYasudaBlood()
+    for gamma_dot in test_shear_rates:
+        mu_python = carreau_yasuda_python(gamma_dot)
+        mu_rust = blood_model.apparent_viscosity(gamma_dot)
+        diff_pct = abs(mu_python - mu_rust) / mu_python * 100
+        print(f"    γ̇ = {gamma_dot:7.0f}: Python {mu_python*1000:7.4f}, Rust {mu_rust*1000:7.4f} mPa·s (Diff: {diff_pct:.4f}%)")
```

Ruff (0.15.9)

[warning] String contains ambiguous `γ` (GREEK SMALL LETTER GAMMA). Did you mean `y` (LATIN SMALL LETTER Y)?

(RUF001)